frontend/src/components/common/navigation/BackButton.tsx

```diff
@@ -7,19 +7,25 @@ import { ArrowBackOutlineIcon } from "../../../assets/icons";
 interface BackButtonProps {
   size?: string;
   text?: string;
+  onClick?: () => void;
   returnTo?: string;
 }
 
 const BackButton = ({
+  onClick,
   returnTo,
-  size = "sm",
-  text = "Back",
+  size = "xl",
```

Comment:
This doesn't impact anything else, right?

Reply:
yep